### PR TITLE
Fix problem with pip 10 (issue #14)

### PR DESCRIPTION
**Pull request by ebrensi**
This is a proposed fix for issue https://github.com/rbanffy/pip-chill/issues/14

I got a working solution here:
https://stackoverflow.com/questions/49923671/are-there-any-function-replacement-for-pip-get-installed-distributions-in-pip

It works great for me!

### CODE REVIEW

**Comment by ebrensi** [comment]
Travis fails with Python 3.3 build for some reason?

File: pip_chill/pip_chill.py [hunk 1]
```diff
@@ -16,8 +16,8 @@ def chill(show_all=False):
     # Gather all packages that are requirements and will be auto-installed.
     distributions = {}
     dependencies = {}
-
-    for distribution in pip.get_installed_distributions():
+
```
Comment (leorochael) [inline pip_chill/pip_chill.py]: This line added unnecessary whitespace.

File: pip_chill/pip_chill.py [hunk 2]
```diff
@@ -16,8 +16,8 @@ def chill(show_all=False):
     # Gather all packages that are requirements and will be auto-installed.
     distributions = {}
     dependencies = {}
-
-    for distribution in pip.get_installed_distributions():
+    
+    for distribution in pkg_resources.working_set:
```
Comment (leorochael) [inline pip_chill/pip_chill.py]: Replacing `pip.get_installed_distributions` with `pkg_resources.working_set` is a better fix than importing `pip._internal.utils.misc.get_installed_distributions` as it's *internal* as the name say.

However this means this PR should also add `setuptools` to the `requirements` list in `setup.py`

`pip` should be there as well right now but isn't, even though it's actually used in the code, not just in tests.

If `pip` was there, this PR should replace `pip` with `setuptools`.

Comment (jayvdb) [reply pip_chill/pip_chill.py]: Another approach is to use `pip-shims` or `pip-api`
